### DISCUSSION TITLE
Une app bancaire virtuelle

### DISCUSSION THREAD

**Discussion by LamidhoPoulo** [community/community#134719] [Programming Help]
### Select Topic Area

General

### Body

Salut je Suis sur github sa fait pas longtemps je suis nouveau dans le monde code novice nul et je voudrais savoir s'ils ya moyen de intégrée une communauté pour m'apprendre à code j'ai tout fait sur YouTube mais je n'ai rien compris Aidez-moi je serais reconnaissant merci. Bowe

**Comment by Akash1134** [comment]
Hi @LamidhoPoulo , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
